vms/shift/tests/test_viewVolunteerShift.py

```diff
@@ -171,28 +171,6 @@ def test_view_with_assigned_and_unlogged_shift(self):
         self.assertEqual(upcoming_shift_page.get_shift_start_time(), '9 a.m.')
         self.assertEqual(upcoming_shift_page.get_shift_end_time(), '3 p.m.')
 
-    def test_log_hours_and_logged_shift_does_not_appear_in_upcoming_shifts(self):
```

Comment:
The later part of the test is still valid right?  `logged hours not appearing in upcoming shifts`

Reply:
But @Monal5031  I can't log hours for a future shift, so this function doesn't make sense.

Reply:
@abhishekarya286 @Monal5031 and how can I log hours for a future shift?

Reply:
He was just talking about the code after line 188. It implies if you don't sign up for a shift, there should be nothing on the upcoming shifts page.

Reply:
But this test already exists

Reply:
https://github.com/anjali-dhanuka/vms/blob/531853b4c5f8fb5235b819582dab8697b6ad725d/vms/shift/tests/test_viewVolunteerShift.py#L151

Reply:
Yes, then it's fine. I was not aware about this test.